client/view-v2.js

```diff
@@ -2418,6 +2424,9 @@ module.exports = (function () {
           case 'license':
             editNote[p] = formContent.noteLicenseValue ?? edit.note[p]
             break
+          case 'cdate':
+            editNote[p] = formContent.noteCDateValue ?? edit.note[p]
+            break
           case 'pdate':
```

Comment:
constructUpdatedEdit does not have mdate is it intentional?

Reply:
thanks for the catch, added